src/agent/sqlite_manager/tests/sqlite_manager_test.cpp

```diff
@@ -0,0 +1,16 @@
+#include "gtest/gtest.h"
+class SQLiteManagerTest : public ::testing::Test
```

Comment:
Missing line after include, could we use `<>` for this include?

Reply:
Done